### ISSUE TITLE
NVDA spuriously announces baseline when told to read superscripts and subscripts when reading webpages

### ISSUE THREAD

**Issue by ultrasound1372** [nvaccess/nvda#11383]
### Steps to reproduce:
Go to a page that chains up links on a line, or that if arrowing down you will encounter a place with a link and then text. For example [the Wikipedia page on electricity](https://en.wikipedia.org/wiki/Electricity), which along with the possible donate header has a series of graphic links at the top denoting that it is a good article and that it is semi-protected.
### Actual behavior:
When reading the series of graphic links at the top, NVDA announces "link    graphic    baseline  This is a good article. Click here for more information.  baseline     link    graphic    baseline  Page semi-protected  baseline"
### Expected behavior:
NVDA should not say baseline at all, or maybe just once at the beginning since I arrowed to that piece of text.
#### NVDA installed/portable/running from source:
installed
#### NVDA version:
2020.2 beta 2
#### Windows version:
Windows 10 1909
#### Name and version of other software in use when reproducing the issue:
I was using Mozilla Firefox (latest stable version) when reading this page, as I generally do. I have not tested if this behavior occurs with other browsers.
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
I have not, as this beta is what added the separate toggle for superscripts and subscripts.
#### If addons are disabled, is your problem still occuring?
Yes
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
No, I do not believe it is applicable here.
### A note about why this may be caused
When I press NVDA+F on the graphic links to report formatting information, it actually says no formatting information. This doesn't make any sense, and this might be the root cause of this particular bug.  
This behavior can also be observed when scrolling down slightly and encountering the enlargeable image with a caption. Upon encountering the graphic link no formatting information is reported, causing it to say baseline again. It does not say it again when you press down arrow to read the text "enlarge," which still has no formatting information. Arrowing down again will then report baseline again as it hits a link and some text, making up the caption of the image.  
Perhaps this is a multilayered fix? Heuristic elimination of repeated baseline announcements without the change to another script, and the failure to retrieve formatting information. On a graphic it makes sense to have no formatting information, but on the text "enlarge" there probably should be.  
I just realized that it is still reporting it on this very page where I submit the issue, if I press b to cycle through buttons and go back and fourth between the browse and submit buttons it reports baseline each time before the button. Arrowing from the submit button up to the markdown styling link also announces it.

**Comment by ultrasound1372** [comment]
Updated to the release candidate, behavior is not changed. This is actually quite aggressive, as even if I take graphic links out of the picture and just work with links and maybe some text, this still happens, even on multiple chunks of text put in with no links in between. For example the text that says when I opened this issue is read like "visited  link    ultrasound1372  baseline   opened this issue   baseline  7 minutes ago  baseline   · 0 comments"  
It just seems that upon any change in formatting or control, moving in and out of edit fields, through headings, links etc, it forgets that it was already at baseline. I cannot use this feature as it stands, which means that I cannot properly read anything in proper scientific notation that isn't using MathML.

**Comment by Adriani90** [comment]
Closing as duplicate of #11078.

**Comment by feerrenrut** [comment]
Just to confirm, if you disable the superscripts and subscripts in the document formatting panel of NVDA settings, then "baseline" is no longer reported.

**Comment by XLTechie** [comment]
@feerrenrut: while I am not the OP, I did test his original case and found it 
valid. Answering your question: yes, turning off super and subs also removes 
"baseline".

The main complaint seems to be, that after a reversion to baseline font from a 
super/subscript, baseline should not keep being announced for each new item.
It should only be announced if we were not previously in baseline, and switched 
back to it.

In other words: it should only be announced once.

**Comment by ultrasound1372** [comment]
Yes. And in the case of most of the top of the aforementioned article, baseline was repeatedly reported yet no text up there even tried to go into su(b/per)script. Same with the github text. As I said, it appears to be either a change in other formatting information, or a change in control, or maybe one just causes the other, that produces this. Formatting attributes change, it forgets that it was already at baseline, and for some reason doesn't just default to assuming that everything is already at baseline. Discussion should probably continue in #11078 though, unless you think this is a better issue to demonstrate this.

**Comment by feerrenrut** [comment]
Yes, thanks @XLTechie for clarifying this further, I understand this can still be improved. My first priority was to confirm whether this was a regression or just a feature that needs further improvement.
